recipes/r-arrow/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = '0.12.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-arrow
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: e93e43343544e344bbc912b89da01d8abf66596f029d26b2b135b102a9e39895
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [win]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - {{posix}}pkg-config
```

Comment:
We can probably get rid nowdays of the `{{posix}}` prefix here as https://github.com/conda-forge/pkg-config-feedstock is now also built for Windows. (related to this feedstock).

Reply:
ok. done